variables.tf

```diff
@@ -309,18 +309,18 @@ variable "gitlab_runner_ssh_cidr_blocks" {
   default     = ["0.0.0.0/0"]
 }
 
-variable "docker_machine_docker_cidr_blocks" {
```

Comment:
FYI, this variable is not being used as the docker machine docker port ingress access is controlled via security group instead of CIDR.

Reply:
Thanks